src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball returnBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        randomBall.setColor(colorSupplier.getRandomColor());
+        Random random = new Random();
+        randomBall.setNumber(random.nextInt(100));
```

Comment:
The random number generation should include 100 as a possible value. Use `random.nextInt(101)` to ensure the range is 0 to 100 inclusive.